python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -298,12 +308,24 @@ def _build_register_command_data(
             software_versions = self._scan_for_software(
                 engine, display_name, icon, versions
             ) or []
+
+            # Information used to construct the equivalent of Desktop "collapse rules"
+            group_name = self._get_group_name(engine)
+            sorted_versions = self._sort_group_versions(
+                [software_version.version for software_version in software_versions]
```

Comment:
For the auto discovery, seems we should return a `software_version` property which can be used for the group name. Do you think that makes sense?